core/actor-zeta/messaging/message_header.hpp

```diff
@@ -17,9 +17,9 @@ namespace actor_zeta {
 
             message_header &operator=(const message_header &) = default;
 
-            message_header(message_header &&) noexcept = default;
+            message_header(message_header &&) = default;
 
-            message_header &operator=(message_header &&) noexcept = default;
+            message_header &operator=(message_header &&) = default;
```

Comment:
What is the reason to remove noexcept ?

Reply:
The reason is MVC compile errors (function signature incompatibility with and without keyword 'noexcept' in implementation). This two lines is my flub. Already fixed in new commit.